benchmarks/dynamo/pr_time_benchmarks/test_check_result/result_test.csv

```diff
@@ -0,0 +1,4 @@
+a, instruction count, 90, 0.01
+b, memory, 200, 0.1
+c, something, 107, 0.1
+d, missing-test, 10
```

Comment:
CSV with variable number of columns? 🤔

Reply:
whats your question? ah there should be only three i will remove 0.01 from the first row. 
this is am using to test my script.